### PR TITLE
Feature/redux fetch

### PR DESCRIPTION
**Pull request by andreasrs**
Demonstrates how to perform simple datafetching using redux-fetcher. Populates component tree using this data.

### CODE REVIEW

File: examples/complex/package.json [hunk 1]
```diff
@@ -6,6 +6,7 @@
   "license": "MIT",
   "roc": "1.0.0-alpha8",
   "dependencies": {
+    "redux-fetcher": "github:vgno/redux-fetcher#master",
```
Comment (DLMR) [inline examples/complex/package.json]: Should we wait to merge this until it has been published and use npm here?

**Comment by DLMR** [comment]
This looks awesome, I love the refactoring that have been done on the code! Will merge when we have closed https://github.com/vgno/redux-fetcher/pull/1.

File: examples/complex/package.json (hunk 1 above)
Comment (andreasrs) [inline examples/complex/package.json]: Let's wait

**Comment by andreasrs** [comment]
It has been published. The latest commit updates the example to use it. Works!

**Pull request merged**